sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/Rules/ForLoopIncrementSignCheck.cs

```diff
@@ -0,0 +1,243 @@
+﻿/*
```

Comment:
In all our analyzer we end the rule implementation name with "Check". But I see that `sonar-dotnet` repository is not using this convention.  There's only 1 other file out of 408 in this directory that has its name ending with "Check.cs". Can you validate with @andrei-epure-sonarsource that we want to use this naming convention here?

Reply:
good catch! it should be `ForLoopIncrementSignTest`

Reply:
Done.